data/os/Debian/9.yaml

```diff
@@ -0,0 +1,3 @@
+---
+corosync::provider: 'crm'
+corosync::pcs_version: NULL
```

Comment:
NULL is a bit uncommon. is that equal to undef in Puppet? in that case we usually use ~

Reply:
`null` is the [canonical YAML value](https://yaml.org/spec/1.2.2/#10211-null) for 'lack of value', e.g. Ruby's `nil`; and Puppet uses `NilClass` as `PUndefType`.

According to [`Psych::ScalarScanner`](https://github.com/ruby/psych/blob/v4.0.3/lib/psych/scalar_scanner.rb#L41), they're both fine.

Personally I'm more for `null` (or more specifically `NULL` as it obviously says "I'm not you're average unquoted string!" and gets fancy highlights), but if `~` is the more popular choice…

Reply:
Looks like `~` may be more well-known because the YAML 1.0 spec only allowed `~` and empty values to represent `null`. I didn't know YAML 1.2 also allowed the word `null` until now. :)